org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -77,7 +77,7 @@ class Translator {
                 var hashFromPayload = this._generateHmac(verification.key, payloadAsString);
 
                 if (hashFromWink !== hashFromPayload) {
-                    throw new OpenT2TError(401, "Notification signature doesn't match.  Expecting " + hashFromWink + " and calculated " + hashFromPayload);
+                    throw new OpenT2TError(401, OpenT2TConstants.HMacSignatureVerificationFailed);
```

Comment:
i see --- maybe log the error with teh specifics (though without logging hte input logging the hash seems useless also) for debugging help/telemetry) but we can tackle that later.